### PR TITLE
open-vm-tools: 11.1.5 -> 11.2.0

### PR DESCRIPTION
**Pull request by CheariX**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

The two patch files could be removed since they are now include in the original code. The remaining change is a simple version bump.

### CODE REVIEW

**Comment by CheariX** [comment]
Any update on this?

File: pkgs/applications/virtualization/open-vm-tools/default.nix [hunk 1]
```diff
@@ -23,11 +23,6 @@ stdenv.mkDerivation rec {
   buildInputs = [ fuse glib icu libdnet libmspack libtirpc openssl pam procps rpcsvc-proto xercesc ]
       ++ lib.optionals withX [ gdk-pixbuf-xlib gtk3 gtkmm3 libX11 libXext libXinerama libXi libXrender libXrandr libXtst ];
 
-  patches = [
-    ./recognize_nixos.patch
```
Comment (SuperSandro2000) [inline pkgs/applications/virtualization/open-vm-tools/default.nix:27]: This patch can't be dropped as it isn't upstreamed yet.

Comment (CheariX) [reply pkgs/applications/virtualization/open-vm-tools/default.nix:27]: Well... actually it works flawlessly. I'm using it for 2 weeks without any issues.
Earlier versions of `open-vm-tools` needed this patch and could be built without it.

I think, the reason is, that newer version of `open-vm-tools` now parse [/etc/os-release](https://github.com/vmware/open-vm-tools/blob/d0b7c90917f4f2afcb0583988d58bc3ec75ad023/open-vm-tools/lib/misc/hostinfoPosix.c#L1614) and used the infos contained therein.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 103466` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>open-vm-tools</li>
    <li>open-vm-tools-headless</li>
  </ul>
</details>

**Pull request merged**